notifications/notifications.go

```diff
@@ -0,0 +1,93 @@
+package notifications
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// Sender contains the configuration information to send events to the notification service
+type Sender struct {
+	URL string
+}
+
+// EmailMessage contains the required fields for formatting email messages
+type EmailMessage struct {
+	Subject string `json:"subject"`
+	Body    string `json:"body"`
+}
+
+// SlackMessage contains the required fields for formatting slack messages
+type SlackMessage struct {
+	Text string `json:"text"`
+}
+
+// BrowserMessage contains the required fields for formatting browser notifications
+type BrowserMessage struct {
+	Type      string    `json:"type"`
+	Text      string    `json:"text"`
```

Comment:
Why both: text and message, for plain text and HTML?

Reply:
Confused. The BrowserMessage does not have a `message` field

Reply:
I mean `Message` from line 33 here

Reply:
Oh whoops! Nice catch